lib/console.js

```diff
@@ -349,7 +349,7 @@ Console.prototype.table = function(tabularData, properties) {
   if (properties !== undefined && !ArrayIsArray(properties))
     throw new ERR_INVALID_ARG_TYPE('properties', 'Array', properties);
 
-  if (tabularData == null || typeof tabularData !== 'object')
+  if (tabularData === null || typeof tabularData !== 'object')
```

Comment:
It's not obvious, but this significantly changes the semantics of the check in a non-backwards compatible way.

```js
null == null; // true
undefined == null; // true
null === null; // true
undefined === null; // false
```

The current check makes sure that `tabularData` is also not undefined.

Reply:
But the later can check it, typeof undefined === undefined so typeof tabularData !== 'object' is true

Reply:
This is 100 % subjective, but maybe this would be a bit more Obviously Correct™ if the conditions were switched?

Reply:
Just not seeing the point of the change then if it simply shifts the `undefined` case a bit to the right.

Reply:
Just change to strict equal to unify the overall code style.
If there is any problems, feel free to close it.😜

Reply:
Nah, I'm not overly happy with it but I won't block it :-)